config/activity_labels/recipe_coffee.yaml

```diff
@@ -0,0 +1,101 @@
+version: "1"
+labels:
+  # Item:
+  # - id: Integer identification number of the label.
+  # - label: Short human-readable semantic string to succinctly represent the
+  #          label.
+  # - step_detail: Longer descriptive string detailing the labeled step. This
+  #                currently acts as the step "instruction," but it's presence
+  #                here is questionable.
+  - id: 0
+    label: "background"
+    step_detail: "Background"
+  - id: 1
+    label: "measure-12oz-water"
+    step_detail: "Measure 12 ounces of water in the liquid measuring cup"
+  - id: 2
+    label: "pour-water-kettle"
+    step_detail: >-
+      Pour the water from the liquid measuring cup into the electric
+      kettle
+  - id: 3
+    label: "turn-kettle-on"
+    step_detail: "Turn on the Kettle"
+  - id: 4
+    label: "place-dipper-on-mug"
+    step_detail: "Place the dripper on top of the mug"
+  - id: 5
+    label: "filter-fold-1"
+    step_detail: >-
+      Take the coffee filter and fold it in half to create a semi-circle
+  - id: 6
+    label: "filter-fold-2"
+    step_detail: "Fold the filter in half again to create a quarter-circle"
+  - id: 7
+    label: "place-filter"
+    step_detail: >-
+      Place the folded filter into the dripper such that the the point of the
+      quarter-circle rests in the center of the dripper
+  - id: 8
+    label: "spread-filter"
+    step_detail: "Spread the filter open to create a cone inside the dripper"
+  - id: 9
+    label: "scale-turn-on"
+    step_detail: "Turn on the kitchen scale"
+  - id: 10
+    label: "place-bowl-on-scale"
+    step_detail: "Place a bowl on the scale"
+  - id: 11
+    label: "zero-scale"
+    step_detail: "Zero the scale"
+  - id: 12
+    label: "measure-coffee-beans"
+    step_detail: "Add coffee beans to the bowl until the scale reads 25 grams"
+  - id: 13
+    label: "pour-coffee-grinder"
+    step_detail: "Pour the measured coffee beans into the coffee grinder"
+  - id: 14
+    label: "set-timer-20"
+    step_detail: "Set timer for 20 seconds"
+  - id: 15
+    label: "timer-turn-on"
+    step_detail: "Turn on the timer"
+  - id: 16
+    label: "grind-beans"
+    step_detail: >-
+      Grind the coffee beans by pressing and holding down on the black part of
+      the lid
+  - id: 17
+    label: "pour-beans-filter"
+    step_detail: >-
+      Pour the grounded coffee beans into the filter cone prepared in step 2
```

Comment:
Remove reference to step 2

Reply:
This was a raw copy-paste of the sentences from the steps doc, but I believe this will be moot as I will be removing the step detail field here as mentioned in chat.